dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -87,7 +88,16 @@ public class DataxTask extends AbstractTask {
      * todo: Create a shell script to execute the datax task, and read the python version from the env, so we can support multiple versions of datax python
      */
     private static final String DATAX_PYTHON = Optional.ofNullable(System.getenv("DATAX_PYTHON")).orElse("python2.7");
-
+    /**
+     * datax on yarn jar path
+     * https://github.com/duhanmin/datax-on-yarn
+     */
+    private static final String dataxOnYarnJar = System.getenv("DATAX_ON_YARN_JAR");
+    /**
+     * example: HADOOP_OPTS="-Xms32m -Xmx128m" /usr/bin/yarn
+     */
+    private static final String YARN_BIN =
+            StringUtils.isEmpty(System.getenv("YARN_BIN")) ? "yarn" : System.getenv("YARN_BIN");
```

Comment:
It's better to put this to `dolphinscheduler_env.sh` and docs so that users can use it more easily.